xls/contrib/integrator/ir_integrator.cc

```diff
@@ -18,6 +18,122 @@
 
 namespace xls {
 
+absl::StatusOr<std::unique_ptr<IntegrationFunction>>
+IntegrationFunction::MakeIntegrationFunctionWithParamTuples(
+    Package* package, absl::Span<const Function* const> source_functions) {
+  // Create integration function object.
+  std::unique_ptr<IntegrationFunction> integration_function =
```

Comment:
nit: no need for stuttering the type with make_unique. Ie:

auto integration_function = absl::make_unique...

Reply:
Done.